src/App/OrganisasjonerOgTilgangerProvider.tsx

```diff
@@ -153,22 +149,7 @@ export const OrganisasjonerOgTilgangerProvider: FunctionComponent = (props) => {
         undefined
     );
     const { addAlert } = useContext(AlertContext);
-    useEffect(() => {
-        measureAll(
-            (tidMs) => {
-                amplitude.logEvent('komponent-lastet', {
-                    komponent: 'OrganisasjonerOgTilgangerProvider',
-                    tidMs,
-                });
-            },
-            hentAltinnTilgangssøknader()
-                .then(setAltinnTilgangssøknader)
-                .catch((error) => {
-                    Sentry.captureException(error);
-                    setAltinnTilgangssøknader([]);
```

Comment:
I den gamle koden: hvis `hentAltinntilgangssøknader` feilet, så ble `altinnTilgangssøknader` satt til `[]`, fordi `undefined` representerte "laster-tilstanden". Og når den er `undefined`, så blokkerer det rendringen av resten av siden. Det i seg selv er nok noe vi burde gå bort fra, men det blir en større jobb tror jeg.

Nå returneres `undefined` fra `useAltinnTilgangssøknader` hvis kallet feiler. Her tror jeg det er bedre å returnere `[]` ved feil. Og jeg tenker faktisk vi burde droppe retries på denne. I hvert fall frem til den er flyttet ut av den "kritiske pathen" for å laste siden.

Reply:
endret nå. Jeg la til litt caching via useEffect for å hindre infinite loop. I dette tilfellet inne i hooken. Var allerede så mye som skjedde i provideren df100b0d1f9ad27a3fef62f9bb49405f51791c7c